proto/axelar/nexus/v1beta1/types.proto

```diff
@@ -42,3 +43,22 @@ message LinkedAddresses {
   axelar.nexus.exported.v1beta1.CrossChainAddress recipient_address = 2
       [ (gogoproto.nullable) = false ];
 }
+
+message RateLimit {
+  string chain = 1
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
+  cosmos.base.v1beta1.Coin limit = 2 [ (gogoproto.nullable) = false ];
+  google.protobuf.Duration window = 3
+      [ (gogoproto.stdduration) = true, (gogoproto.nullable) = false ];
+}
+
+message TransferRate {
+  string chain = 1
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
+  cosmos.base.v1beta1.Coin amount = 2 [ (gogoproto.nullable) = false ];
+  uint64 epoch = 3;
+  bool outgoing = 4; // indicates whether the tracking is for transfers outgoing
+                     // to that chain or incoming from it
```

Comment:
use an enum instead?

Reply:
not sure why i convinced myself to go with a bool lol